OOVPADatabase/D3D8LTCG.OOVPA.inl

```diff
@@ -163,18 +168,20 @@ OOVPATable D3D8LTCG_OOVPAV2[] = {
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_StencilFail, 1024, 1036, 1048, 1060, 1072, 1084),
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_TextureFactor, 1036),
 	REGISTER_OOVPAS(D3DDevice_SetRenderState_TwoSidedLighting, 1024, 1036, 1048, 1060, 1072, 1084),
-	REGISTER_OOVPAS(D3DDevice_SetRenderState_VertexBlend, 1024, 1036),
+	REGISTER_OOVPAS(D3DDevice_SetRenderState_VertexBlend, 1024, 1036, 1048),
```

Comment:
And still having failed to compile, D3DDevice_SetRenderState_VertexBlend_1048 is also missing. Hopefully, this is the last one.